Themes/default/languages/Help.english.php

```diff
@@ -625,6 +625,10 @@
 
 $helptxt['field_reg_require'] = 'If this field is required during registration, it will also be required on profile changes.';
 
+$helptxt['translatable_fields'] = 'If you wish to provide multi-language support for your custom profile fields you can use translatable tokens on both the name and description fields.<br>
+Anything enclosed with braces {} will be considered as a token and will be matched against a $txt string, for example, the token: {gender} will be replaced by the value of $txt[\'gender\']<br>
+It is <strong>very important</strong> to note you need to add the matched $txt string to <strong>ALL</strong> currently installed languages, it is recommended to put them on index.language.php for the translation to be available across your forum.';
```

Comment:
This is a perfect opportunity to direct the admin to the built-in language editor, so that admins who are reluctant to edit files directly can do the task using a nice, friendly UI instead.

```suggestion
$helptxt['translatable_fields'] = 'If you wish to provide multi-language support for your custom profile fields you can use translatable tokens on both the name and description fields.<br><br>
Anything enclosed with braces <span class="monospace">{}</span> will be considered as a token and will be matched against a $txt string. For example, the token <span class="monospace">{gender}</span> will be replaced by the value of $txt[\'gender\']<br><br>
It is <strong>very important</strong> to add the matched $txt string to <strong>ALL</strong> currently installed languages. To do this, go to the <a href="%1$s?action=admin;area=languages;sa=edit;%3$s=%2$s" class="bbc_link" target="_blank" rel="noopener">Edit Languages</a> page and complete the following steps for each installed language:
<ol class="bbc_list" style="list-style-type: decimal">
	<li>Click the Edit button next to the language name.</li>
	<li>In the "Select entries to edit" menu, choose "Modifications".</li>
	<li>Scroll to bottom and click the "Add Another Item" button.</li>
	<li>Follow the prompts to add your new string.</li>
</ol>';
```